src/components/welcome/Welcome.jsx

```diff
@@ -84,35 +77,46 @@ const getSteps = (withSelector) => [
             component="h2"
             sx={{ flex: "80%", marginTop: "8px" }}
           >
-            Questions Game
+            {t("helper.welcome.page2.title")}
           </Typography>
         </Box>
         <Box
           sx={{ display: "flex", flexDirection: { xs: "column", md: "row" } }}
         >
           <Box>
             <Typography component="p" sx={{ mt: 2 }}>
-              Does the food product belong to this brand?
-            </Typography>
-            <Typography component="p" sx={{ mt: 2 }}>
-              Answer simply with a <strong>yes/no</strong>! Don't know the
-              answer? That's alright, just skip it.
+              {t("helper.welcome.page2.text1")}
             </Typography>
+            <Typography
+              component="p"
+              sx={{ mt: 2 }}
+              dangerouslySetInnerHTML={{
+                // https://github.com/i18next/react-i18next/issues/189#issuecomment-235181562
+                __html: t("helper.welcome.page2.text2"),
```

Comment:
What about using `<Trans/>` https://react.i18next.com/latest/trans-component#using-with-react-components ?

Reply:
I did not want to use the Trans element for consistency reasons, but from your words, I take it that you are OK with the change